src/main/resources/static/js/results.js

```diff
@@ -0,0 +1,50 @@
+function exportToCSV() {
+
+    var url = "/export/";
+
+    //get the selected values to export CSV
+    var selectedVals = $("div.card-body input:checked").next('label').map(function () {
```

Comment:
Might better to give that card-body an ID because otherwise, this select is unclear. Could also cause issues if we had another card-body with inputs.